mud-contracts/world/src/modules/smart-gate/systems/SmartGateSystem.sol

```diff
@@ -150,7 +150,6 @@ contract SmartGateSystem is EveSystem, AccessModified {
       revert SmartGate_GateNotLinked(sourceGateId, destinationGateId);
     }
     SmartGateLinkTable.set(sourceGateId, destinationGateId, false);
-    SmartGateLinkTable.set(destinationGateId, sourceGateId, false);
```

Comment:
why is this line removed ?

Reply:
A branch merge mistake. Good catch!

Reply:
fixed in commit 6d7f21bfb9ff1ed3565a4c68b7bb29811cc2a75e